### PR TITLE
Updated resource file

### PR DESCRIPTION
**Pull request by jyvenugo**
Adding string for Loading Package failed and 2 other strings for the wxs file,
Remove and 
Warning - removing this will cause the following app(s) to be unremovable: <insert app name>. These app(s) should be removed before proceeding.

### CODE REVIEW

File: preview/MsixCore/msixmgr/msixmgr.rc [hunk 1]
```diff
@@ -96,17 +125,32 @@ BEGIN
...
+    IDS_STRING_HELP_OPTION_UNPACK_APPLYACLS 
+                            "optional parameter that applies ACLs to the resulting package folder(s) and their parent folder"
+    IDS_STRING_HELP_OPTION_UNPACK_VALIDATESIGNATURE 
+                            "optional parameter that validates a package's signature file before unpacking the package. This will require that the package's certificate is installed on the machine."
+    IDS_STRING_HELP_OPTION_APPLYACLS 
+                            "Applies ACLs to a package folder (an unpacked package)"
+    IDS_STRING_HELP_OPTION_APPLYACLS_PACKAGEPATH 
+                            "the path to the folder to apply acls to"
+    IDS_STRING_LOADING_FAILED "Loading Package Failed"
+    IDS_STRING_WXS_REMOVE   " Remove"
```
Comment (wcheng-msft) [inline preview/MsixCore/msixmgr/msixmgr.rc]: >   IDS_STRING_WXS_REMOVE   " Remove" [](start = 1, length = 36)

if it's just "Remove", we don't need this localized.

File: preview/MsixCore/msixmgr/msixmgr.rc [hunk 2]
```diff
@@ -123,21 +167,3 @@ BEGIN
 /////////////////////////////////////////////////////////////////////////////
 #endif    // not APSTUDIO_INVOKED
 
-/////////////////////////////////////////////////////////////////////////////
-//
-// Version
-//
-#define VER_FILEVERSION 0,0,0,0
-#define VER_FILEVERSION_STR "0.0.0.0"
```
Comment (wcheng-msft) [inline preview/MsixCore/msixmgr/msixmgr.rc]: keep these variables

File: preview/MsixCore/msixmgr/msixmgr.rc (hunk 1 above)
Comment (jyvenugo) [reply preview/MsixCore/msixmgr/msixmgr.rc]: Won't Remove be different in different languages?

Comment (wcheng-msft) [reply preview/MsixCore/msixmgr/msixmgr.rc]: Wix already has localization for "Remove", "Install" "Select the operation you wish to perform" etc all those strings in the dialog already; 
we just need localization for any non-standard custom strings we added.

---
In reply to: [337798141](https://github.com/microsoft/msix-packaging/pull/242#discussion_r337798141) [](ancestors = 337798141)

Comment (jyvenugo) [reply preview/MsixCore/msixmgr/msixmgr.rc]: Removed remove.

File: preview/MsixCore/msixmgr/msixmgr.rc (hunk 2 above)
Comment (jyvenugo) [reply preview/MsixCore/msixmgr/msixmgr.rc]: I did not change these at all. I think its something because of my compiler.

Comment (jyvenugo) [reply preview/MsixCore/msixmgr/msixmgr.rc]: These still exist, the order for these got switched to a different line because of adding a new string.

**Comment by jyvenugo** [comment]
Closing this PR. Created new PR - https://github.com/microsoft/msix-packaging/pull/243 with feedback from code review comments.

**Pull request closed without merging**